File: Domain/Recarga.cs
namespace Domain
{
    public class Recarga
    {
        public Guid Id { get; set;}

         public string title { get; set; }
        public string price { get; set; }
        public string description { get; set; }
        public string category { get; set; }

        public virtual ICollection<UserRecarga> UserRecargas {get; set;}

    }
}

File: client/my-app/src/app/models/recarga.ts
import { IUser } from "./user";

export interface IRecarga{

    id: string;
    title: string;
    price: string;
    description: string;
    category: string;
    attendees: IAttendee[];
}

export interface IAttendee {
    username : string;
    displayName: string;
    image: string;
    isHost: boolean;
    state: string;
    refunded: boolean;
    dateBought: Date;
    appUserId: string;
    appUser: IUser
    recarga: IRecarga
}